### PR TITLE
Fixed support of remote webdriver

### PR DESCRIPTION
**Pull request by quarckster**

### CODE REVIEW

**Comment by oshtaier** [comment]
Any context for these changes? Any test executed using remote driver and results for it?

**Comment by quarckster** [comment]
I used to run selenium tests locally in a separate vnc session using remote webdriver. Robotello doesn't seem to work in that way. I tried to run this test https://github.com/RedHatInsights/insights-tests/blob/master/src/tests/satellite/ui/test_rhai.py#L104 locally and it passed.

**Comment by oshtaier** [comment]
Once more, after your changes, robottelo code start to work? Or it is still doesn' work?

**Comment by quarckster** [comment]
Yes, Robottelo now can work via remote webdriver. See excerpt of my robotello.properties:
```ini
...
browser=selenium
webdriver=remote
webdriver_desired_capabilities=platform=Linux,browserName=chrome,unhandledPromptBehavior=dismiss,unexpectedAlertBehaviour=ignore,acceptInsecureCerts=True
...
```

File: robottelo/config/base.py [hunk 1]
```diff
@@ -1079,6 +1079,7 @@ def __init__(self):
         self.webdriver = None
         self.webdriver_binary = None
         self.webdriver_desired_capabilities = None
+        self.command_executor = None
```
Comment (ldjebran) [inline robottelo/config/base.py]: Please update robottelo.properties.sample with this new option

**Comment by oshtaier** [comment]
@quarckster ping

File: robottelo/config/base.py (hunk 1 above)
Comment (quarckster) [reply robottelo/config/base.py]: Done

**Pull request merged**